extensions/elytron-security-oauth2/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-build-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../../build-parent/pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-elytron-security-oauth2-parent</artifactId>
+    <name>Quarkus Security OAuth2</name>
```

Comment:
This does not follow the convention for artifact names (see all the others)

Reply:
I don't understand, it's the parent pom of the quarkus-elytron-security-oauth2 extension so it follows the same convention as the other extension I checked (quarkus-elytron-security, quarkus-elasticsearch-rest-client, ...)

Reply:
OK, will use `Quarkus - Security OAuth2`

Reply:
Yes, that's it.

Reply:
done